### PR TITLE
kernel: add ip virtual server kernel modules and ipvsadm tool to manipulate ip_vs kernel tables

### PR DESCRIPTION
**Pull request by ezplanet**
IPVS (IP Virtual Server) implements transport-layer load balancing inside the Linux kernel, so called Layer-4 switching. IPVS running on a host acts as a load balancer at the front of a cluster of real servers, it can direct requests for TCP/UDP based services to the real servers, and makes services of the real servers to appear as a virtual service on a single IP address.

This change adds kmod-nf-ipvs and kmod-nf-conntrack-ftp kernel modules options to LEDE kernel netfilter and ipvsadm kernel tool to manipulate ip_vs kernel tables.

Signed-off-by: Mauro Mozzarelli mauro@ezplanet.org

### CODE REVIEW

File: package/network/utils/ipvsadm/Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipvsadm
+PKG_VERSION:=1.29
+PKG_MAINTAINER:=Mauro Mozzarelli <mauro@ezplanet.org>
+PKG_LICENSE:=GPL-1.0
+PKG_RELEASE:=2
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.kernel.org/pub/linux/utils/kernel/ipvsadm/
```
Comment (hauke) [inline package/network/utils/ipvsadm/Makefile]: Please use 
`` PKG_SOURCE_URL:=@KERNEL/linux/utils/kernel/ipvsadm/ ``

File: package/network/utils/ipvsadm/patches/001-Makefile.patch [hunk 2]
```diff
@@ -0,0 +1,75 @@
+diff -Naur a/libipvs/Makefile b/libipvs/Makefile
```
Comment (hauke) [inline package/network/utils/ipvsadm/patches/001-Makefile.patch]: Will you send this patch to the upstream project so they can integrate it?

File: package/network/utils/ipvsadm/Makefile [hunk 3]
```diff
@@ -16,7 +16,7 @@ PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.kernel.org/pub/linux/utils/kernel/ipvsadm/
-PKG_SHA256SUM:=c3de4a21d90a02c621f0c72ee36a7aa27374b6f29fd4178f33fbf71b4c66c149
+PKG_HASH:=c3de4a21d90a02c621f0c72ee36a7aa27374b6f29fd4178f33fbf71b4c66c149
```
Comment (hauke) [inline package/network/utils/ipvsadm/Makefile]: Please squash this into the previous commit

**Comment by ezplanet** [comment]
Thank you for your comments.
I am not sure how to remove the merge from this PR, I would appreciate if you could provide guidance.
Last resort it to delete and start over ...

**Comment by ezplanet** [comment]
Deleting the repository and starting over to tidy up code

**Pull request closed without merging**